service/apiserver/internal/logic/account/getaccountlogic.go

```diff
@@ -104,9 +108,23 @@ func (l *GetAccountLogic) GetAccount(req *types.ReqGetAccount) (resp *types.Acco
 				Balance: asset.Balance.String(),
 				Price:   strconv.FormatFloat(assetPrice, 'E', -1, 64),
 			})
+
+			// BNB for example:
+			//   1. Convert unit of balance from wei to BNB
+			//   2. Calculate the result of (BNB balance * price per BNB)
+			balanceInFloat := new(big.Float).SetInt(asset.Balance)
+			unitConversion := new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil)
```

Comment:
Different assets have different units. `Decimal` field of asset is for this, i believe.